plugins/plugin-php/che-plugin-php-lang-server/src/main/resources/files/default_php_content

```diff
@@ -1,5 +1,3 @@
-// Hello World program
-
 <?php
-  echo "Hello World!";
-?>
+
+echo "Hello World!\n";
```

Comment:
Absolutely. Check here: http://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag